_damo_deprecated.py

```diff
@@ -80,84 +80,91 @@ def check_output(*popenargs, **kwargs):
     # If a regions of a size >=2MiB keeps small access frequency for >=100ms,
     # avoid the region using huge pages (call madvise() with MADV_NOHUGEPAGE).
     2M      max     0       25      100ms   max nohugepage
-'''
+"""
+
 
 def fields_to_v0_scheme(fields):
     scheme = _damon.Damos()
     scheme.access_pattern = _damon.DamosAccessPattern(
-            sz_bytes = [_damo_fmt_str.text_to_bytes(fields[0]),
-                _damo_fmt_str.text_to_bytes(fields[1])],
-            nr_accesses = [_damo_fmt_str.text_to_percent(fields[2]),
-                _damo_fmt_str.text_to_percent(fields[3])],
-            nr_accesses_unit = _damon.unit_percent,
-            age = [_damo_fmt_str.text_to_us(fields[4]),
-                _damo_fmt_str.text_to_us(fields[5])],
-            age_unit = _damon.unit_usec)
+        sz_bytes=[_damo_fmt_str.text_to_bytes(fields[0]), _damo_fmt_str.text_to_bytes(fields[1])],
+        nr_accesses=[
+            _damo_fmt_str.text_to_percent(fields[2]),
+            _damo_fmt_str.text_to_percent(fields[3]),
+        ],
+        nr_accesses_unit=_damon.unit_percent,
+        age=[_damo_fmt_str.text_to_us(fields[4]), _damo_fmt_str.text_to_us(fields[5])],
+        age_unit=_damon.unit_usec,
+    )
     scheme.action = fields[6].lower()
     return scheme
 
+
```

Comment:
I'd prefer one line between functions.

Reply:
I'd like to have some time to think about this.  Let's discard black at the moment, if you don't mind.